session/bootstrap.go

```diff
@@ -142,7 +142,7 @@ const (
 		Table_name	CHAR(64),
 		Column_name	CHAR(64),
 		Timestamp	TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-		Column_priv	SET('Select','Insert','Update'),
+		Column_priv	SET('Select','Insert','Update','References'),
```

Comment:
These changes will only take effect for new installs. Please write a bootstrap task to `ALTER TABLE` on existing installs :-)

Reply:
So a `upgradeToVerXX` should be written for the `ALTER TABLE`?

Reply:
Yes, that's right.

Reply:
Added.